content/intro-to-storybook/vue/es/test.md

```diff
@@ -0,0 +1,151 @@
+
+### Iniciando Git
+
+Primero tienes que configurar Git para tu proyecto en el directorio local. Chromatic usa el historial de Git para hacer un seguimiento de los componentes de tu interfaz de usuario.
+
+```bash
+$ git init
+```
+
+Luego agrega archivos al primer commit.
+
+```bash
+$ git add .
+```
+
+Ahora haz commit de los archivos.
+
+```bash
+$ git commit -m "taskbox UI"
+```
+
+### Añadiendo Chromatic
+
+Agregando el paquete como una dependencia.
+
+```bash
+yarn add --dev storybook-chromatic
+```
+
+Importa Chromatic en tu archivo `.storybook/config.js`.
+
+```javascript
+import { configure } from '@storybook/vue';
+import 'storybook-chromatic';
+
+import '../src/index.css';
+
+const req = require.context('../src', true, /.stories.js$/);
+function loadStories() {
+  req.keys().forEach(filename => req(filename));
+}
+
+configure(loadStories, module);
+```
+
+Ahora [logueate en Chromatic](https://www.chromaticqa.com/start) con tú cuenta de GitHub (Chromatic solo te pedirá algunos permisos simples). Crea un proyecto con nombre "taskbox" y copia tu `app-code` único.
+
+<video autoPlay muted playsInline loop style="width:520px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/chromatic-setup-learnstorybook.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Ejecuta el comando de prueba en la línea de comandos para configurar las pruebas de regresión visual para Storybook. No olvides añadir tu código de aplicación único en el `<app-code>`.
+
+```bash
+./node_modules/.bin/chromatic test --app-code=<app-code>
+```
+
+<div class="aside">
+<code>--do-not-start</code> es una opción que le dice a Chromatic que no inicie Storybook. Usa esto si ya tienes a Storybook corriendo. Si no, omite el <code>--do-not-start</code>.
+</div>
+
+Una vez el primer test esté completo, tenemos líneas base de prueba para cada historia. En otras palabras, capturas de cada historia que ya se conocen como "buenas". Los futuros cambios a estas historias serán comparados con estás lineas base.
+
+![Chromatic baselines](/intro-to-storybook/chromatic-baselines.png)
+
+## Capturando un cambio en la interfaz de usuario
+
+La prueba de regresión visual se basa en la comparación de imágenes del nuevo código de la interfaz de usuario renderizado con las imágenes de la línea base. Si se detecta un cambio en la interfaz de usuario, se notificará. Vea cómo funciona ajustando el fondo del componente `Task`:
+
+![code change](/intro-to-storybook/chromatic-change-to-task-component.png)
+
+Esto produce un nuevo color de fondo para el artículo.
+
+![task background change](/intro-to-storybook/chromatic-task-change.png)
+
+Usa el comando de prueba anterior para ejecutar otra prueba cromática.
```

Comment:
If you don't mind update the wording a bit here, it's not a chromatic test, but a test with the package that was added in this section, note that the use of a capital "C" that indicates the usage of the product.